doc/cephfs/cephfs-shell.rst

```diff
@@ -14,10 +14,8 @@ executed command. In non-interactive mode, cephfs-shell issues a command and
 exits right after the command's execution is complete with the command's return
 value.
 
-Behaviour of CephFS Shell can be tweaked using cephfs-shell.conf. CephFS Shell
-looks for it by default at the path provided in environment variable
-`CEPHFS_SHELL_CONF` and then in user's home directory
-(`~/.cephfs-shell.conf`).
+Behaviour of CephFS Shell can be tweaked using ``cephfs-shell.conf``. Refer to
+the section on ``cephfs-shell.conf`` on this page for details.
```

Comment:
Add hyperlink to `cephfs-shell.conf` section
```suggestion
Behaviour of CephFS Shell can be tweaked using ``cephfs-shell.conf``.
 ```

Reply:
Yeah, that's much better. Thanks!